src/site/content/en/podcasts/index.njk

```diff
@@ -0,0 +1,271 @@
+---
+layout: layout
+title: Podcasts
+description: |
+  web.dev is the ultimate resource for developers of all backgrounds to learn,
+  create, and solve on the web.
+date: 2018-11-05
+---
+<div class="about-landing-page">
+  <header class="w-masthead-about">
+    <div class="w-masthead-about__container">
+      <div class="w-masthead-about__left">
+        <h1 class="w-headline--two w-font-weight--medium w-mt--non">
+          Podcasts
+        </h1>
+        <p>
+          Level up by listening to podcasts from Google about the web industry.
```

Comment:
Yay! Copy thoughts:

"Level up by listening to podcasts from Google about the web industry." -->
"Level up your web development skills by listening to podcasts from Google Developers."

Reason: gives a context to what you are "Leveling up"

"These offer you" --> "Our podcasts highlight"

I'd remove "and greatest" because it's subjective

"level up your web development skills" --> "learn what's new and how to build with best practices in mind"
(Since we already said Level Up at the start)

So all together:
"Level up your web development skills by listening to podcasts from Google Developers. Our podcasts highlight the latest in modern web development with pro-tips, insights, and techniques to help you learn what's new and how to build with best practices in mind"

Reply:
![Screen Shot 2020-03-17 at 3 54 18 PM](https://user-images.githubusercontent.com/1134620/76909105-90e3f480-6867-11ea-8709-62c7f6f86102.png)

edits made with visbug hehe, lgtm!